### PR TITLE
Spec fixes

### PR DESCRIPTION
**Pull request by AACO**
Changes to spec, fixed issue with assign gear

### CODE REVIEW

File: description.ext [hunk 1]
```diff
@@ -124,58 +124,58 @@ class CfgRespawnTemplates {
...
-    class F_Radios {
-        class F_Radios {
-            file = "f\radios";
-            class reinitRadio {};
-            class acreRadioSetup {};
-            class acreRadioSetPlayerChannels {};
-        };
+  };
+  class F_Radios {
+      class F_Radios {
```
Comment (MikeMatrix) [inline description.ext]: Indentation.

File: f/respawn/respawn_gui_code.sqf [hunk 2]
```diff
@@ -1,313 +1,304 @@
...
-        _groupNameArray = respawnMenuGroupNames select _i;
-        _control lbAdd (_groupNameArray select 0);
+  disableSerialization;
+  deadPlayerList = [];
+  selectedRespawnGroup = [];
+
+  //Faction selection control
+  _control = ((findDisplay 26893) displayCtrl 26894);
+  {
+    _factionArray = respawnMenuFactions select _forEachIndex;
```
Comment (MikeMatrix) [inline f/respawn/respawn_gui_code.sqf]: Why not use `_x` here?

File: f/respawn/respawn_gui_code.sqf [hunk 3]
```diff
@@ -1,313 +1,304 @@
...
 
 fn_truncateDecimal = {
-    params ["_decimal"];
-    _decToString = str _decimal;
-    _splitDec = _decToString splitString ".";
-    _splitDec select 0;
+  params ["_decimal"];
+  _decToString = str _decimal;
+  _splitDec = _decToString splitString ".";
+  _splitDec select 0;
```
Comment (MikeMatrix) [inline f/respawn/respawn_gui_code.sqf]:
```
params ["_decimal"];
((str _decimal) splitString ".") select 0
```
This has less variable access time, minor improvement, not sure if needed.

File: f/respawn/respawn_gui_code.sqf [hunk 4]
```diff
@@ -1,313 +1,304 @@
...
-    _entry = (selectedRespawnGroup select _selection);
-    _entry set [2, ((_entry select 2) + 1 ) % (count respawnMenuRoles)];
-    selectedRespawnGroup set [_selection, _entry];
-    
-    [] call fn_update_aliveListBox;
+  disableSerialization;
+  _deadListBox = ((findDisplay 26893) displayCtrl 26891);
+  _groupListBox = ((findDisplay 26893) displayCtrl 26892);
+
+  _selection =  (lbCurSel _groupListBox);
```
Comment (MikeMatrix) [inline f/respawn/respawn_gui_code.sqf]: 2 spaces after `=`

File: f/respawn/respawn_gui_code.sqf [hunk 5]
```diff
@@ -1,313 +1,304 @@
...
-    _entry = (selectedRespawnGroup select _selection);
-    _entry set [0, ((_entry select 0) + 1) % 7];
-    selectedRespawnGroup set [_selection, _entry];
-    
-    [] call fn_update_aliveListBox;
+  disableSerialization;
+  _deadListBox = ((findDisplay 26893) displayCtrl 26891);
+  _groupListBox = ((findDisplay 26893) displayCtrl 26892);
+
+  _selection =  (lbCurSel _groupListBox);
```
Comment (MikeMatrix) [inline f/respawn/respawn_gui_code.sqf]: 2 spaces after `=`

File: f/spect/fn_CamInit.sqf [hunk 6]
```diff
@@ -2,203 +2,184 @@
...
 // params
-_this spawn {
-  _unit = [_this, 0, player, [objNull]] call BIS_fnc_param;
-  _oldUnit = [_this, 1, objNull, [objNull]] call BIS_fnc_param;
-  _forced = [_this, 4, false, [false]] call BIS_fnc_param;
+_seagull = [_this, 0, objNull, [objNull]] call BIS_fnc_param;
+_deadUnit = [_this, 1, player, [objNull]] call BIS_fnc_param;
+_respawnType = [_this, 2, 3, [3]] call BIS_fnc_param;
+_respawnDelay = [_this, 3, 3, [3]] call BIS_fnc_param;
+_forced = [_this, 4, false, [false]] call BIS_fnc_param;
```
Comment (MikeMatrix) [inline f/spect/fn_CamInit.sqf]: This can be replaced with `params`

File: description.ext (hunk 1 above)
Comment (AACO) [inline description.ext]: Matching file style (for now) not going to update

**Pull request merged**